chalice/cli/__init__.py

```diff
@@ -79,11 +80,14 @@ def cli(ctx, project_dir, debug=False):
 @click.option('--port', default=8000, type=click.INT)
 @click.option('--stage', default=DEFAULT_STAGE_NAME,
               help='Name of the Chalice stage for the local server to use.')
+@click.option('--autoreload', is_flag=True)
```

Comment:
Seems like it should be turned on by default so people don't need to discover the functionality, it just magically works. We could flip it and let them disable it if they want to for some reason. `--no-autoreload`. Thoughts?

Reply:
It makes sense.